compiler/rustc_const_eval/src/interpret/intern.rs

```diff
@@ -291,7 +291,7 @@ pub enum InternKind {
 /// tracks where in the value we are and thus can show much better error messages.
 /// Any errors here would anyway be turned into `const_err` lints, whereas validation failures
 /// are hard errors.
-#[tracing::instrument(level = "debug", skip(ecx))]
+#[instrument(level = "debug", skip(ecx))]
```

Comment:
I'd prefer if the tracing in the interpreter used the `trace` level. Currently we make `debug` already so drowned in noise that we basically lose the entire point of even having that distinction.